src/main/java/com/example/travelservice/services/TripServiceImpl.java

```diff
@@ -44,4 +44,10 @@ public List<WeatherDto> showTripWeather(Trip trip) {
 
     }
 
+    @Override
+    public Trip updateTrip(Long id, Trip trip) {
+        trip.setId(id);
```

Comment:
You need to find first trip with such id

Reply:
fixed